tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
We agreed yesterday to only support docker-pool on systems that had it precreated.  These tests all fail since they are creating new thin-pools.  The only other way to make these pass is to hard code it in the case that the user does not specify a any arguments, or specified docker-storage-setup as arg1, which is what you rejected.